Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM node:14-alpine AS test
+FROM node:16-alpine AS test
 
 WORKDIR /envelope-game
 
 COPY package*.json /envelope-game/
 
-RUN npm install 
+RUN yarn install 
```

Comment:
we should copy the `yarn.lock` before running `yarn install`

Reply:
we figured running install would generate a new lock file every time but this makes sense